examples/multisig/virtual.eg.ts

```diff
@@ -31,13 +32,28 @@ const { alexa, billy, carol, david } = await createDevUsers()
 /// With this state, we can hydrate from / use an existing virtual multisig.
 /// Let's see if we can hydrate the virtual multisig state from command line arguments.
 /// If we haven't specified any virtual multisig state, we deploy a new virtual multisig.
-let { state } = parse(Deno.args, { string: ["state"] })
+const args = parse(Deno.args, { string: ["state"], boolean: ["use-stash"] })
+let state = args.state
+const useStash = args["use-stash"]
```

Comment:
Instead of creating the stash proxy in this example (is `use-stash` is specified), should we accept a stash address as a string flag and use that? We could also create a `create_stash.eg.ts` example to showcase usage of the proxy pallet. Might be cleaner to keep them separate.

Reply:
> const { state, "use-stash": useStash } = parse(Deno.args, { string: ["state"], boolean: ["use-stash"] })

state has to be mutable here but we can do this for `use-stash`